tutorials/hoon/landscape-tile.md

```diff
@@ -0,0 +1,276 @@
+
+Run `npm run build`. This copies files into the pier that you linked into the `/app` directory of your desk. Your Hoon file will be `/app/testing.hoon`, and your js file will be `/app/testing/js/tile.js`.
+
+![](https://media.urbit.org/docs/landscape-tile/npm-run-build.png)
+
+Starting with Urbit version 0.8.0, files are no longer automatically synced from your Unix pier to your ship, so after running the build command, you'll need to run the `|commit %home` command in your ship's Dojo.
+
+![](https://media.urbit.org/docs/landscape-tile/commithome.png)
+
+Once this is done, you can start the application by running `|start %testing` in your ship's Dojo. Then navigate to  `http://localhost:80` (or the appropriate port), and log in with the password you get from running `+code` in the Dojo. Then you should see where you should see your new tile.
+
+Any time you make changes to the Hoon file or the js file, you will need to run the `npm run build` command in the repo as well as `|commit %home` in the Dojo. Ford will then automatically rebuild your application with the new changes.
+
+## Our Own Tile
+
+Let's examine a sample tile that has different Hoon code from the default, to explore how the tile system works. This program, composed of the `testing.hoon` and `tile.js` files, creates a tile which can be used to send `hi` messages to ships, just the way you can with `|hi ~zod` from the dojo. Keep in mind that unless you boot other fakeships you won't get responses from ships other than `~zod`.
+
+`tile.js` does the actual front-end rendering using React. The details of React are left for the reader to learn on their own, but you can see here how to store data in the state. That tile will be rendered every time the state gets updated. Calling `api.action` will send a JSON poke to our gall app. We have attached the function where we use that to the button on the tile, so it will get run when the button is clicked.
+
+In your pier, replace the code in `/app/testing.hoon` with the Hoon code below.
+
+``` hoon
+/+  *server
+/=  tile-js
+  /^  octs
+  /;  as-octs:mimes:html
+  /:  /===/app/testing/js/tile
+  /|  /js/
+      /~  ~
+  ==
+=,  format
+|%
++$  move  (pair bone card)
++$  poke
+  $%  [%launch-action [@tas path @t]]
+      [%helm-hi @t]
+  ==
++$  card
+  $%  [%poke wire dock poke]
+      [%http-response =http-event:http]
+      [%connect wire binding:eyre term]
+      [%diff %json json]
+  ==
+--
+|_  [bol=bowl:gall ~]
+++  bound
+  |=  [wir=wire success=? binding=binding:eyre]
+  ^-  (quip move _+>.$)
+  [~ +>.$]
+++  prep
+  |=  old=(unit ~)
+  ~&  'it built'
+  ^-  (quip move _+>.$)
+  =/  launcha
+    [%launch-action [%testing /testingtile '/~testing/js/tile.js']]
+  :_  +>.$
+  :~
+    [ost.bol %connect / [~ /'~testing'] %testing]
```

Comment:
Is using prefix `~` for non-shipnames a landscape thing? Seems likely to get confusing if someone brings up a `simple-walrus` app(letter swap away from valid @p)

Reply:
all landscape applications have `~` prefixes in their endpoint